packages/amplify_datastore/example/lib/main.dart

```diff
@@ -50,6 +50,8 @@ class _MyAppState extends State<MyApp> {
   String _postWithIdNotEquals = '';
   String _firstPostFromResult = '';
   String _allPostsWithoutRating2Or5 = '';
+  String _streamingData = '';
+  Stream<SubscriptionEvent<Post>> stream = null;
```

Comment:
[`Stream`s](https://dart.dev/tutorials/language/streams#two-kinds-of-streams)! Cool.